web/packages/operations/src/alarm.ts

```diff
@@ -234,10 +251,12 @@ export const initializeAlarms = async () => {
 
     let client = new CloudWatchClient({})
     let cloudWatchAlarms = []
-    let alarmCommandSharedInput = {
+    let alarmCommandSharedInput: any = {
         Namespace: CLOUD_WATCH_NAME_SPACE + "-" + name,
-        Threshold: 0,
-        TreatMissingData: "breaching",
+        Threshold: 0
+    }
+    if(name == "polkadot_mainnet") {
+        alarmCommandSharedInput.TreatMissingData = "breaching";
     }
```

Comment:
For Westend we config the `SCAN_INTERVAL` as 4 hours and not breaching missing data to suppress alarms.

Reply:
+1.

Why do we get missing data in westend though?

Reply:
`SCAN_INTERVAL=240`(i.e. 4 hour) for Westend.